packages/fast-form-generator-react/src/form/form-item.align-horizontal.style.ts

```diff
@@ -0,0 +1,57 @@
+import { Direction, ellipsis, localizeSpacing, toPx } from "@microsoft/fast-jss-utilities";
+import { applyInputStyle, applyLabelStyle, applyWrapperStyle } from "../utilities/form-input.shared-style.style";
```

Comment:
i think you can just call the file name form-input.style instead of adding shared-style

Reply:
sure